public/angular/js/controllers/sources/index.js

```diff
@@ -10,7 +10,7 @@ angular.module('Aggie')
   'Socket',
   function($scope, $rootScope, flash, sources, Source, Tags, Socket) {
     $scope.sources = sources;
-
+    console.log($scope.sources);
```

Comment:
is this for testing or production?

Reply:
Fixed in https://github.com/TID-Lab/aggie/commit/556591795b099aa5367e620965e5268e40adac7a

It was for testing.